packages/node-cli/package.json

```diff
@@ -19,6 +19,8 @@
 	"repository": "https://github.com/oracle/netsuite-suitecloud-sdk/tree/master/packages/node-cli",
 	"main": "./src/suitecloud.js",
 	"preferGlobal": true,
+	"sdkFileName": "cli-2022.2.1.jar",
```

Comment:
Maybe change this to sdkFilename to be consistent with what we have in the config file?